iot/http_example/cloudiot_http_example.js

```diff
@@ -141,16 +141,35 @@ function publishAsync (authToken, messageCount, numMessages) {
       binary_data: binaryData
     }
   };
+
+  // Custom backoff functions
+  function retryNotSuccess (err, response, body) {
```

Comment:
Ditto to below comment - can these come from a library?

Reply:
Done, switched to recommended library that uses built-in exponential backoff.